### ISSUE TITLE
executeWithSigner() with FileContentsQuery throws HWC Error: WalletConnect error

### ISSUE THREAD

**Issue by wasinnimlaor** [hashgraph/hedera-sdk-js#1726]
### Description

![image](https://github.com/hashgraph/hedera-sdk-js/assets/62790229/5668506c-3618-41cd-827c-bb8a7c8fce6a)

Hi, I'm trying to get the file contents from HFS with `FileContentsQuery().executeWithSigner()` but it throws `HWC Error: WalletConnect error`

It `FileContentsQuery()` works fine with `execute(client)`, but since I want my DApp to connect with user's wallet to let the user to be a signer for transactions, I have to execute transaction with `signer` instead of `client`.

Also, I've tried execture `FileInfoQuery()` with `signer` and it works fine. So, I think the issue is about executing `FileContentsQuery` with `signer`

### Steps to reproduce

1. Connect DApp with hedera chain wallet to get signer
2. Create file and send message to HCS
3. Call `FileContentsQuery` with `executeWithSigner`

### Hedera network

testnet

### Version

2.29.0

### Operating system

None

**Comment by petreze** [comment]
Hey @wasinnimlaor 

I was able to execute successfully both `FileContentsQuery` and `FileInfoQuery`. Can you provide an example of your exact setup? Also, you are trying to execute it directly from the FE, right?

**Comment by wasinnimlaor** [comment]
@petreze Yes, I execute from FE. Did you execute `FileContentsQuery` with `executeWithSigner` method?

FYI: I create the signer with `@bladelabs/blade-web3.js`

![image](https://github.com/hashgraph/hedera-sdk-js/assets/62790229/047096ec-4d6d-405d-b5c8-18a145e13c0b)

**Comment by petreze** [comment]
@wasinnimlaor 
Yes but in BE directly, now I'm trying your approach and setup
How do you build this `signer` from `blade-web3.js`?

**Comment by wasinnimlaor** [comment]
![image](https://github.com/hashgraph/hedera-sdk-js/assets/62790229/0e62f74d-8b96-4111-8526-7300f6758f64)

![image](https://github.com/hashgraph/hedera-sdk-js/assets/62790229/84d8c69d-a8d4-4870-b4b5-c7e49f354e91)

![image](https://github.com/hashgraph/hedera-sdk-js/assets/62790229/a2aeda77-0eeb-4560-9dc1-9831a9f5152c)


Hey @petreze, I build the signer in App.tsx. I create the connector on first landing and then create a signer.

**Comment by petreze** [comment]
Hey @wasinnimlaor 
I was able to reproduce the issue entirely. To wrap up, by using the `BladeWallet` as `signer` I can execute `FileInfoQuery` but `FileContentsQuery` fails with the same errors. But as previously mentioned I was able to execute bot queries from a BE without any errors, so I assumed that the issue is in how you create the `BladeWallet` connection. The strange thing here is that with exactly the same setup of the blade wallet one query succeeds and the other one fails. Did you consider to open an issue in their GH.
I will continue to try and find the root cause
